source/partials/_footer.html.erb

```diff
@@ -0,0 +1,7 @@
+<footer>
+    <ul class="footer">
+        <li class="footer__link"><%= link_to '860-406-1021', 'tel:#{8604061021}' %></li>
```

Comment:
I would shy away from providing your phone number. It's not the safest, most conventional way for someone to reach out to you from the internets. I think email/LinkedIn would suffice. Curious to hear your thoughts though!

Reply:
Maybe that's why I've been getting so much spam...

But yeah, good point - I figure I can replace it with Github & Dribble in it's place.